### ISSUE TITLE
Data List - Radio Button List - Clear selection button sets value to "null" as a string

### ISSUE THREAD

**Issue by DaltonKirk** [leekelleher/umbraco-contentment#245]
### Which Contentment version are you using?

4.0.0

### Which Umbraco version are you using? For example: 8.14.1 - don't just write v8

10.0.1

### Bug summary

I have a DataList using a Radio Button List as the list editor and a User-defined List as the data source.

On the Radio Button List List Editor I have enabled Allow Clear.

I noticed when I click this button to clear the selection and save it. I then get a string value of "null" saved against that property. Which was getting past my string.IsNullOrEmpty checks.

### Steps to reproduce

1. Create a DataList using a Radio Button List as the list editor and a User-defined List as the data source.
![image](https://user-images.githubusercontent.com/22774653/183640894-4522fa4e-c09e-40c1-ada9-cc5c203b7f86.png)

2. Add it to something where you can use it.
3. Click the 'Clear selection' button and save your object.
![image](https://user-images.githubusercontent.com/22774653/183641037-3b4e7553-613d-46ad-be51-ccd0a0224d76.png)

4. Check the value that is saved against your property it will be "null"

### Expected result / actual result

I would expect this to be an empty string "" and not a string with the value of "null"

### Do you have Umbraco ModelsBuilder enabled?

- [X] Yes, it is enabled.

**Comment by leekelleher** [comment]
Hi @DaltonKirk, thanks for raising this. I can reproduce the bug on Umbraco 10. However the frustrating thing is that I can't reproduce it on Umbraco 8, (of which Contentment is currently supporting both 🤦).

If I set the value as either `null` or `undefined`, then the value will be stored in the database as `"null"` (a string of the word "null").  I'm unsure whether something changed in the Content saving workflow of Umbraco 10, but then it's not a bug I can fix from within Contentment, so I guess my best option is to set the value as an empty string (`""`).

Fixed in commit e5c7bede1ce5aef9ba8c8b49a7a667326e35a672. _(I don't have a date for a patch release yet)_
